docs/usage.md

```diff
@@ -269,6 +269,30 @@ With this configuration in place deployment should happen automatically every ti
 
 There is also a [Shiny server application](https://posit.co/download/shiny-server/), which you can install on your own infrastruture and use to host applications yourself.
 
+## Gene set enrichment analysis
+
+Currently, two tools can be used to do gene set enrichment analysis.
+
+### GSEA
+
+[GSEA](https://www.gsea-msigdb.org/gsea/index.jsp) tests for differential genes from within a user-provided set of genes; this requires a GMT or GMX file. The following example shows how to enable this:
+
+```bash
+--gsea_run true \
+--gene_sets_files gene_sets.gmt
+```
+
+### g:Profiler
+
+The [gprofiler2](https://cran.r-project.org/web/packages/gprofiler2/vignettes/gprofiler2.html) package can be used to test which pathways are enriched in the sets of differential genes produced by the the DESeq2 or limma modules. It is an R interface for the g:Profiler webtool. In the simplest form, this feature can be enabled with the parameters from the following example:
+
+```bash
+--gprofiler2_run true \
+--gprofiler2_organism mmusculus
+```
+
+Either a token from a previous g:profiler run, a custom GMT file or the organism (mmusculus for Mus musculus, hsapiens for Homo sapiens etc.) is required. Check the [pipeline webpage](https://nf-co.re/differentialabundance/1.2.0/parameters#gprofiler2) for the parameters for this step. Also, by default the analysis will be run with a background list of genes that passed the abundance filter (i.e. those genes that actually had some expression); see for example https://doi.org/10.1186/s13059-015-0761-7 for why this is advisable. You can provide your own background list with `--gprofiler2_background_file background.txt`; alternatively, if you want to not use any background, set this: `--gprofiler2_background_file false`.
```

Comment:
So I did set the original priorities intentionally (first the specific/reproducible options token and custom GMT, then the organism which just queries whichever database versions are the current ones). But update the module to put the GMT file as last instance.

Reply:
Done. I had to do some fiddling to keep the error message if multiple GMT files are provided. I'll some time update the module again so that the pipeline can be run with multiple GMT files like GSEA